CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Read our [guidelines for writing a good changelog entry](https://github.com/biom
 ## Unreleased
 
 ### Analyzer
+
+#### Enhancements
+
+- Add a code action to replace `rome-ignore` with `biome-ignore`. Use `rome check --apply-unsafe` to update all the comments. The action is not bulletproof, and it might generate unwanted code, that's why it's unsafe action.
```

Comment:
```suggestion
- Add a code action to replace `rome-ignore` with `biome-ignore`. Use `biome check --apply-unsafe` to update all the comments. The action is not bulletproof, and it might generate unwanted code, that's why it's unsafe action.
```

Just wondering: have you example where the fix generates unwanted code?

Reply:
Check the updated snapshots